ca_candidates/people.py

```diff
@@ -0,0 +1,205 @@
+                            p.add_link(social)
+                elif name == 'Nimâ Machouf':
+                    socials = candidatepage.xpath('//section[@class="BioSite-components-BodySections-Socials-Socials__socials--vRiFp"]/div/a/@href')
+                    for social in socials:
+                        p.add_link(social)
+                elif name == 'Tammy Bentz':
+                    candidatecontact = self.lxmlize(url + "/contact/")
+                    email = candidatecontact.xpath('//a[contains(@href, "mailto:")]/@href')[0].replace("mailto:", "")
+                    phone = candidatecontact.xpath("//li[contains(text(), 'Call')]/text()")[0].replace("Call ", "")
+                    p.add_contact("email", email)
+                    p.add_contact("voice", phone, "office")
+                    socials = candidatecontact.xpath('//li[contains(@class, "wp-social-link")]/a/@href')
+                    for social in socials:
+                        p.add_link(social)
+                elif name == 'Bhutila Karpoche':
+                    contact = candidatepage.xpath('//address[@class="contact-block-items flex flex-justify-center"]/div')
+                    phone = contact[1].xpath('./a/@href')[0].replace("tel:", "")
+                    address = contact[2].xpath('./div/text()')[1].strip()
+
+                    p.add_contact("voice", phone, "office")
+                    p.add_contact("address", address, "office")
+
+                    links = candidatepage.xpath('//ul[@class="connect-items list-unstyled engage-block-connect-items engage-block-item-content content-font-size"]')[0]
+                    socials = links.xpath('./li/a/@href')
+                    for social in socials:
+                        if "facebook.com" in social or "youtube.com" in social or "instagram.com" in social or "x.com" in social:
+                            p.add_link(social)
+                elif name == 'Samantha Green':
+                    contact = candidatepage.xpath('//address[@class="contact-block-items flex flex-justify-center"]/div')
+                    phone = contact[1].xpath('./a/@href')[0].replace("tel:", "")
+                    address = contact[2].xpath('./div/text()')[1].strip()
+                  
+                    p.add_contact("voice", phone, "office")
+                    p.add_contact("address", address, "office")
+
+                    socials = candidatepage.xpath('//li[@class="connect-item icons-block-item connect-block-item"]/a/@href')
+                    for social in socials:
+                        p.add_link(social)    
+            else:
+                info = info[1]
+                # getting email, phone number, social media links
+                contacts = info.xpath('./ul/li/a/@href')
+                # check if contacts is empty, if so it probably means they also have a phone number so query separately
+                if len(contacts) == 0:
+                    phone_and_address = info.xpath('./div')[0]
```

Comment:
@zilinweng can you check the if statement above in my most recent scrape this is generating an index out of range error